alerting/src/main/resources/com/amazon/opendistroforelasticsearch/alerting/alerts/alert_mapping.json

```diff
@@ -1,5 +1,4 @@
 {
-  "dynamic": "strict",
```

Comment:
We shouldn't remove this because we don't want fields to be added dynamically if they are not explicitly mentioned in the `alert_mapping`. If we want to support additional information here, let's update the mapping so that it reflects the format we'll allow.